.github/workflows/zip.yml

```diff
@@ -116,7 +116,7 @@ jobs:
         LEGACY: true
       run: SAMPLE="$SDK" TARGET="${SDK}Example" scripts/setup_quickstart_framework.sh \
                                                "${HOME}"/ios_frameworks/Firebase/FirebaseRemoteConfig/* \
-                                               "${HOME}"/ios_frameworks/Firebase/FirebaseAnalytics/FirebaseCore.xcframework \
```

Comment:
I'm surprised this line can be deleted.

Reply:
Looks like FirebaseCore is still needed here based on the [failed job](https://github.com/firebase/firebase-ios-sdk/runs/5882134634?check_suite_focus=true)
Will try another run after adding `FirebaseCore.framework`.

Reply:
Also, the reason other Quickstarts do not have `FirebaseCore.xcframework` is because their have imported `FirebaseAnalytics/*` which includes FirebaseCore framework.